.ci/run.sh

```diff
@@ -71,3 +71,6 @@ sudo -E PATH="$PATH" bash -c "make test"
 
 echo "INFO: Checking log files"
 check_log_files
+
+echo "INFO: Running data collection script"
+sudo -E PATH="$PATH" chronic kata-collect-data.sh
```

Comment:
I think you know what I'm going to say ... ;-)
Does this currently come out in the main CI log? Any chance you can redirect the output (at least stdout) to a known log file location - and then we can use that output in the teardown script, and save:
- injecting the rather large collect script output into the main console log
- having to run the script more than once?

Reply:
Chronic discards all output *unless* the command fails so although there are 2 calls, this one won't bloat the console logs unless the script itself is broken (which we would want to know about).

Reply:
ah, indeed, I didn't spot the `chronic` - mea culpa.

Reply:
I know what you're saying, but the two calls are for different reasons - this one basically checks the script "works" (very simplistically of course), whereas the other call captures the script output if a PR fails (implicitly expecting the script to work).

That teardown script is a bit fragile imho as it has to cater for two separate scenarios so I'd be a little bit leery of adding more complexity to it by getting it to find the output of this script and potentially chop it into bits, etc.